pkg/horizon/volumes.go

```diff
@@ -96,3 +96,22 @@ func getVolumeMounts(
 	}
 	return vm
 }
+
+// GetLogVolumeMount - Horizon API LogVolumeMount
+func GetLogVolumeMount() corev1.VolumeMount {
+	return corev1.VolumeMount{
+		Name:      logVolume,
+		MountPath: "/var/log/horizon",
+		ReadOnly:  false,
+	}
+}
```

Comment:
It would be easier to return a slice of `corev1.VolumeLogMount` here. Then you can just call it from within the pod definition above without having to append the result to a slice.


```suggestion
func GetLogVolumeMount() []corev1.VolumeMount {
	return []corev1.VolumeMount{
		{
			Name:      logVolume,
			MountPath: "/var/log/horizon",
			ReadOnly:  false,
		},
	}
}

```

Reply:
Going to leave it as is for now - if we feel strongly to make this change, we can make another patch.

Looking to get this patch merged asap